Proton/Sources/Swift/Core/LayoutManager.swift

```diff
@@ -410,7 +427,7 @@ class LayoutManager: NSLayoutManager {
         let attributes = lineNumberAttributes(lineNumberFormatting: lineNumberFormatting)
         let text = NSAttributedString(string: "\(lineNumber)", attributes: attributes)
         let markerSize = text.boundingRect(with: .zero, options: [], context: nil).integral.size
-        var markerRect = self.rectForLineNumbers(markerSize: markerSize, rect: rect, width: gutterWidth)
+        let markerRect = self.rectForLineNumbers(markerSize: markerSize, rect: rect, width: gutterWidth)
```

Comment:
There was a warning for this variable not being changed after assignment, so I changed it to be a `let` constant